transbot/models/pygithub_extension.py

```diff
@@ -0,0 +1,65 @@
+try:
+    import github
+    import github.GithubObject.NotSet as NotSet
+except ImportError:
+    github = None
+    NotSet = None
+from hashlib import sha1
+import base64
+
+
+def __get_file_sha1(file_content):
+    return sha1("blob %s\0%s" % (len(file_content), file_content)).hexdigest()
+
+
+def put_file_contents(self, path, message, content, sha=None,
+                      ref=NotSet):
```

Comment:
Could you mention in the docstring that this is a [monkey patch](https://en.wikipedia.org/wiki/Monkey_patch), at first glance, the self outside of a class looked odd.